exercises/clock/.meta/generator/clock_case.rb

```diff
@@ -9,27 +9,31 @@ def name
   end
 
   def workload
-    property == 'equal' ? compare_clocks : simple_test
+    if property == 'create'
```

Comment:
There are 4 known properties in the canonical-data you probably should list them all and raise an error if you find an unexpected one.

Reply:
We also need tests for "sub"/`:-`

These should use the - operator rather than being adding negative minutes.

`assert_equal "10:03", (clock1 - Clock.new(minute: 3)).to_s`